From ad2c20200fbcc2c759ac2fda97eec28aabac0c74 Mon Sep 17 00:00:00 2001
From: Luca Boccassi <bluca@debian.org>
Date: Tue, 3 Jan 2023 23:18:43 +0100
Subject: [PATCH] hwdb: remove spurious whitespace

Fixes test failures.
Follow-up for c3950f0a27fb7d2298a2182f21d645c7017f9c0c and
https://github.com/systemd/systemd/pull/25908

(cherry picked from commit de712a85ffb7ea129536b4e14a1f5cb48f7116f7)

Related: RHEL-5950
---
 hwdb.d/60-sensor.hwdb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index c9d1a2cb90..aa827f97c5 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -910,7 +910,7 @@ sensor:modalias:acpi:KIOX000A*:dmi:*:svnTECLAST:pnX80Pro:*
 # Teclast X89 (E7ED), "tPAD" is too generic also match on BIOS date
 sensor:modalias:acpi:SMO8500*:dmi:*bd12/19/2014:*:rvnTECLAST:rntPAD:*
  ACCEL_MOUNT_MATRIX=1, 0, 0; 0, -1, 0; 0, 0, 1
- 
+
 # Teclast X98 Air 3G (C5J6), "tPAD" is too generic also match on newest BIOS date
 sensor:modalias:acpi:SMO8500*:dmi:*bd05/05/2015:*:rvnTECLAST:rntPAD:*
  ACCEL_MOUNT_MATRIX=1, 0, 0; 0, -1, 0; 0, 0, 1
